src/styles/index.scss

```diff
@@ -1,3 +1,126 @@
 body {
+  font-family: Roboto, sans-serif;
   margin: 0;
+  padding: 0;
+
+  --color-for-main-text: #060B35;
+  --color-for-secondary-text: #616070;
+  --color-for-button: #00ACDC;
+}
+
+.product-card {
+  max-width: 200px;
+  padding: 33px 19px 16px;
+  box-sizing: border-box;
```

Comment:
move this rule to `*` selector

Reply:
Still not fixed

Reply:
I tried to move this code to * selecetor, but all my styles were crashed.